### PR TITLE
Fixes #55 and Fixes #56

### LINKED ISSUE #55: Create json for blog items
Similar to the [current conditions carousels](https://github.com/DOI-USGS/vizlab-home/blob/main/src/assets/content/FlowTiles.js), reformat the structure of [the blog panel](https://github.com/DOI-USGS/vizlab-home/blob/main/src/components/WhatsNew.vue) to be json-driven. This will make it easier to update.

### LINKED ISSUE #56: Update README
Include explanation of how to:
- Add new interactives
- Add new carousels for current conditions
- Add new events in current conditions carousels
- Update blog section. First, https://github.com/DOI-USGS/vizlab-home/issues/55 should be addressed

### PR DESCRIPTION
**Pull request by maggiejaenicke**
Changes made:
-----------
Reformats structure of blog panel to use WhatsNew.js, Adds March streamflow tile, adds instructions to README for updating carousels and blog items.

Closes #55 and closes #56. 

Testing:
----------------------------
Before making this pull request, I:
- [ ] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [ ] Chrome
- [ ] Safari
- [X] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

File: src/assets/content/WhatsNew.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+export default {
+    blogListItems: [
+        {
+            id: '5',
+            date: '07/21/2023:',
+            title: 'Jazz up your ggplots!',
+            url: 'https://waterdata.usgs.gov/blog/ggplot-jazz/',
+            img_src: 'https://waterdata.usgs.gov/blog/static/ggplot-extensions/ggplot-extensions-banner.png',
+            img_alt: 'Musical notes drawn with the center of the notes containing data viz example'
+        },
```
Comment (cnell-usgs) [inline src/assets/content/WhatsNew.js:10]: Moving forward we can include all of this information for each entry. Within the component we can just show the first one, or maybe down the line we'll come up with a way to feature more than one.

File: src/components/WhatsNew.vue [hunk 2]
```diff
@@ -28,94 +28,22 @@
...
-              <a
-                href="https://waterdata.usgs.gov/blog/water-data-science-2022/"
-                target="_blank"
-              >
-                <span class="arrow">
-                  Read &#8594;
-                </span>
-              </a>
-            </li>
-            <!-- end blog list item -->
```
Comment (cnell-usgs) [inline src/components/WhatsNew.vue:118]: So happy to see this cleaned up!

**Review by cnell-usgs — APPROVED** [review]
This looks great. It's a major improvement to the code organization and making it easier to update. Pinging @elmeraa-usgs for awareness - the README updates here can help you add new river conditions and flow-tiles to the portfolio. Let's test them out with the most recent editions.

**Comment by maggiejaenicke** [comment]
@cnell-usgs you'll have to merge this since I don't have write access!

**Pull request merged**